crates/bevy_pbr/src/lib.rs

```diff
@@ -133,13 +135,25 @@ pub struct PbrPlugin {
     pub prepass_enabled: bool,
     /// Controls if [`DeferredPbrLightingPlugin`] is added.
     pub add_default_deferred_lighting_plugin: bool,
+    /// Controls if GPU [`MeshUniform`] building is enabled.
+    ///
+    /// This requires compute shader support.
+    pub using_gpu_uniform_builder: bool,
```

Comment:
I was thinking that we have `WGPU_SETTINGS_PRIO=webgl` (or however you spell it) to opt in to the WebGL 2 baseline, so I was trying to follow the same principle to allow for testing of the WebGL 2 path in non-WebGL 2. The problem is that I don't have the `wgpu` settings handy at the time I'm setting up the plugins in the App, so I went with a bool instead.

I don't really love the solution I came up with either. Suggestions are definitely welcome.

Reply:
I don’t think it should be cfg. It should use feature detection or so.

Reply:
I see how this code was updated for the PR. Looks fine to me.

Reply:
I see how this code was updated in the PR. It looks ok, but ideally we want to move toward using runtime feature detection so that we can deploy bevy that will use WebGPU if available, else WebGL2. But, there are many parts of our code that don't do this yet, so I don't think this usage of cfg is reason to block merging.

Reply:
I made this use feature detection, which was needed to unbreak `WGPU_SETTINGS_PRIO="webgl2"` anyway. Yay!